### PR TITLE
miniflux: 2.0.14 -> 2.0.15 using buildGoModule

### PR DESCRIPTION
**Pull request by dywedir**
[changelog](https://github.com/miniflux/miniflux/releases/tag/2.0.15)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

cc @rvolosatovs @benpye

### CODE REVIEW

File: pkgs/servers/miniflux/default.nix [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
   };
 
-  goDeps = ./deps.nix;
+  modSha256 = "1i3xzl6kkpl4v1229rhg61j1952qxzbhav7fb2hv85903rkz51x1";
 
   doCheck = true;
 
   buildFlagsArray = ''
-    -ldflags=-X ${goPackagePath}/version.Version=${version}
+    -ldflags=-X miniflux.app/version.Version=${version}
```
Comment (kalbasit) [inline pkgs/servers/miniflux/default.nix]: Are you sure that this works? It's must match the full import path of the version package.

**Review by kalbasit — CHANGES_REQUESTED** [review]
@GrahamcOfBorg build miniflux

**Comment by kalbasit** [comment]
@GrahamcOfBorg build miniflux

File: pkgs/servers/miniflux/default.nix (hunk 1 above)
Comment (kalbasit) [reply pkgs/servers/miniflux/default.nix]: I just noticed that it does match the previous goPackagePath. Please double check on your end as I'm afk. The rest of the PR looks good.

Comment (dywedir) [reply pkgs/servers/miniflux/default.nix]: Yep, works fine
```sh
result/bin/miniflux --version
2.0.15
```

**Review by kalbasit — APPROVED** [review]

**Pull request merged**